### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by jonaprieto**
This is a proposal to simplify the README file for newcomers. It's a simpler and concise description of Juvix. Contain also a few examples, better installation instruction, among other things.

### CODE REVIEW

**Review by cwgoes — APPROVED** [review]
ACK, thank you! :rocket:

File: README.md [hunk 1]
```diff
@@ -1,143 +1,198 @@
...
+
+Prerequisites
+--------------
+
+* Install [Stack](https://haskellstack.org):
 
   - For Ubuntu        : `apt install stack`
   - For Debian        : `apt install haskell-stack`
   - For Arch Linux    : `pacman -S stack`
   - For macOS : `brew install haskell-stack`
```
Comment (cyberglot) [inline README.md:40]: misaligned?

File: README.md [hunk 2]
```diff
@@ -1,143 +1,198 @@
...
+  - [libff](https://github.com/scipr-lab/libff),
+  - [libsecp256k1](https://github.com/bitcoin-core/secp256k1), and
+  - [openssl libssl](https://wiki.openssl.org/index.php/Libssl_API).
+
+
+Installation
+------------
+
+1. To install Juvix, you can download its sources using
+   [Git](http://git-scm.com/) from the [Github
```
Comment (cyberglot) [inline README.md:57]: I don't think we need to mention git cuz if the person has never used git, they will have a hard contributing to Juvix anyway.

File: README.md (hunk 1 above)
Comment (jonaprieto) [reply README.md:40]: Good catch!

Comment (jonaprieto) [reply README.md:40]: Oh wait! do you mean the "For Ubuntu        : " thing or what exactly.

File: README.md (hunk 2 above)
Comment (jonaprieto) [reply README.md:57]: The readme is not just for developers, for contributors, we have this other page, CONTR..

File: README.md (hunk 1 above)
Comment (cyberglot) [reply README.md:40]: macOS was misaligned.

**Pull request merged**